storage/include/libstqe.sh

```diff
@@ -20,7 +20,7 @@
 source $(dirname $(readlink -f $BASH_SOURCE))/../../cki_lib/libcki.sh
 
 STQE_GIT="https://gitlab.com/rh-kernel-stqe/python-stqe.git"
-STQE_STABLE_VERSION=${STQE_STABLE_VERSION:-"f1ddf69b"}
+STQE_STABLE_VERSION=${STQE_STABLE_VERSION:-"stable"}
```

Comment:
Is this change on purpose? The commit or the PR description don't mention anything...

Reply:
My bad. I forgot to mention it.